Apply me after running rename.sh

diff --git a/Makefile.am b/Makefile.am
index a87599d..ec72323 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,102 +24,108 @@ endif
 
 ACLOCAL_AMFLAGS = -I m4
 
-AM_CPPFLAGS = -pthread -DLOOLWSD_DATADIR='"@LOOLWSD_DATADIR@"' -DLOOLWSD_CONFIGDIR='"@LOOLWSD_CONFIGDIR@"'
+# quick and easy for now.
+include_paths = -I${top_srcdir}/common -I${top_srcdir}/wsd -I${top_srcdir}/kit
+
+AM_CPPFLAGS = -pthread -DLOOLWSD_DATADIR='"@LOOLWSD_DATADIR@"' -DLOOLWSD_CONFIGDIR='"@LOOLWSD_CONFIGDIR@"' ${include_paths}
 AM_LDFLAGS = -pthread -Wl,-E
 
 AM_ETAGSFLAGS = --c++-kinds=+p --fields=+iaS --extra=+q -R --totals=yes *
 AM_CTAGSFLAGS = $(AM_ETAGSFLAGS)
 
-shared_sources = ChildSession.cpp \
-                 common/FileUtil.cpp \
+shared_sources = common/FileUtil.cpp \
+                 common/IoUtil.cpp \
+                 common/Log.cpp \
+                 common/Protocol.cpp \
+                 common/Session.cpp \
+                 common/MessageQueue.cpp \
                  common/SigUtil.cpp \
-                 IoUtil.cpp \
-                 Log.cpp \
-                 LOOLProtocol.cpp \
-                 LOOLSession.cpp \
-                 MessageQueue.cpp \
-                 Unit.cpp \
-                 UnitHTTP.cpp \
-                 Util.cpp
-
-loolwsd_SOURCES = Admin.cpp \
-                  AdminModel.cpp \
-                  Auth.cpp \
-                  DocumentBroker.cpp \
-                  LOOLWSD.cpp \
-                  ClientSession.cpp \
-                  FileServer.cpp \
-                  PrisonerSession.cpp \
-                  Storage.cpp \
-                  TileCache.cpp \
+                 common/Unit.cpp \
+                 common/UnitHTTP.cpp \
+                 common/Util.cpp
+
+loolwsd_SOURCES = wsd/Admin.cpp \
+                  wsd/AdminModel.cpp \
+                  wsd/Auth.cpp \
+                  wsd/DocumentBroker.cpp \
+                  wsd/LOOLWSD.cpp \
+                  wsd/ClientSession.cpp \
+                  wsd/FileServer.cpp \
+                  wsd/PrisonerSession.cpp \
+                  wsd/Storage.cpp \
+                  wsd/TileCache.cpp \
                   $(shared_sources)
 
 noinst_PROGRAMS = connect \
                   lokitclient \
                   loolforkit-nocaps
 
-connect_SOURCES = Connect.cpp \
-                  Log.cpp \
-                  LOOLProtocol.cpp \
-                  Util.cpp
+connect_SOURCES = tools/Connect.cpp \
+                  common/Log.cpp \
+                  common/Protocol.cpp \
+                  common/Util.cpp
 
-lokitclient_SOURCES = IoUtil.cpp \
-                      Log.cpp \
-                      LOKitClient.cpp \
-                      LOOLProtocol.cpp \
-                      Util.cpp
+lokitclient_SOURCES = common/IoUtil.cpp \
+                      common/Log.cpp \
+                      tools/KitClient.cpp \
+                      common/Protocol.cpp \
+                      common/Util.cpp
 
-loolforkit_SOURCES = LOOLForKit.cpp \
-                     LOOLKit.cpp \
+loolforkit_SOURCES = kit/ChildSession.cpp \
+                     kit/ForKit.cpp \
+                     kit/Kit.cpp \
                      $(shared_sources)
 
 # build a binary with no caps to help debugging
 loolforkit_nocaps_SOURCES = $(loolforkit_SOURCES)
 
-loolmount_SOURCES = loolmount.c
-
-loolmap_SOURCES = loolmap.c
-
-looltool_SOURCES = LOOLTool.cpp
-
-loolstress_CPPFLAGS = -DTDOC=\"$(abs_top_srcdir)/test/data\"
-loolstress_SOURCES = LOOLStress.cpp \
-                     LOOLProtocol.cpp \
-                     Log.cpp
-
-noinst_HEADERS = Admin.hpp \
-                 AdminModel.hpp \
-                 Auth.hpp \
-                 ChildSession.hpp \
-                 Common.hpp \
-                 DocumentBroker.hpp \
-                 Exceptions.hpp \
-                 FileServer.hpp \
-                 common/FileUtil.hpp \
-                 common/SigUtil.hpp \
-                 IoUtil.hpp \
-                 LibreOfficeKit.hpp \
-                 Log.hpp \
-                 LOKitHelper.hpp \
-                 LOOLKit.hpp \
-                 LOOLProtocol.hpp \
-                 LOOLSession.hpp \
-                 LOOLWebSocket.hpp \
-                 LOOLWSD.hpp \
-                 ClientSession.hpp \
-                 PrisonerSession.hpp \
-                 MessageQueue.hpp \
-                 Png.hpp \
-                 QueueHandler.hpp \
-                 Rectangle.hpp \
-                 Storage.hpp \
-                 TileCache.hpp \
-                 TileDesc.hpp \
-                 TraceFile.hpp \
-                 Unit.hpp \
-                 UnitHTTP.hpp \
-                 UserMessages.hpp \
-                 Util.hpp \
+loolmount_SOURCES = tools/mount.c
+
+loolmap_SOURCES = tools/map.c
+
+looltool_SOURCES = tools/Tool.cpp
+
+loolstress_CPPFLAGS = -DTDOC=\"$(abs_top_srcdir)/test/data\" ${include_paths}
+loolstress_SOURCES = tools/Stress.cpp \
+                     common/Protocol.cpp \
+                     common/Log.cpp
+
+wsd_headers = wsd/Admin.hpp \
+              wsd/AdminModel.hpp \
+              wsd/Auth.hpp \
+              wsd/ClientSession.hpp \
+              wsd/DocumentBroker.hpp \
+              wsd/Exceptions.hpp \
+              wsd/FileServer.hpp \
+              wsd/LOOLWebSocket.hpp \
+              wsd/LOOLWSD.hpp \
+              wsd/PrisonerSession.hpp \
+              wsd/QueueHandler.hpp \
+              wsd/Storage.hpp \
+              wsd/TileCache.hpp \
+              wsd/TileDesc.hpp \
+              wsd/TraceFile.hpp \
+              wsd/UserMessages.hpp
+
+shared_headers = common/Common.hpp \
+                 common/IoUtil.hpp \
+                 common/Log.hpp \
+                 common/Protocol.hpp \
+                 common/Session.hpp \
+                 common/Unit.hpp \
+                 common/UnitHTTP.hpp \
+                 common/Util.hpp \
+                 common/MessageQueue.hpp \
+                 common/Png.hpp \
+                 common/Rectangle.hpp \
+                 common/security.h
+
+kit_headers = kit/ChildSession.hpp \
+              kit/Kit.hpp \
+              kit/LibreOfficeKit.hpp \
+	      kit/KitHelper.hpp
+
+noinst_HEADERS = $(wsd_headers) $(shared_headers) $(kit_headers) \
                  bundled/include/LibreOfficeKit/LibreOfficeKit.h \
                  bundled/include/LibreOfficeKit/LibreOfficeKitEnums.h \
                  bundled/include/LibreOfficeKit/LibreOfficeKitInit.h \
diff --git a/common/IoUtil.cpp b/common/IoUtil.cpp
index 4368af1..646dacb 100644
--- a/common/IoUtil.cpp
+++ b/common/IoUtil.cpp
@@ -27,7 +27,7 @@
 #include <Poco/URI.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "Util.hpp"
diff --git a/common/MessageQueue.cpp b/common/MessageQueue.cpp
index 9a34187..b3a7313 100644
--- a/common/MessageQueue.cpp
+++ b/common/MessageQueue.cpp
@@ -13,7 +13,7 @@
 
 #include <Poco/StringTokenizer.h>
 
-#include <LOOLProtocol.hpp>
+#include <Protocol.hpp>
 #include <Log.hpp>
 #include <TileDesc.hpp>
 
diff --git a/common/Protocol.cpp b/common/Protocol.cpp
index 30f75c6..6cd5455 100644
--- a/common/Protocol.cpp
+++ b/common/Protocol.cpp
@@ -7,7 +7,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "config.h"
 
 #include <cassert>
diff --git a/common/Session.cpp b/common/Session.cpp
index c503af7..36bf79c 100644
--- a/common/Session.cpp
+++ b/common/Session.cpp
@@ -7,7 +7,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-#include "LOOLSession.hpp"
+#include "Session.hpp"
 #include "config.h"
 
 #include <sys/stat.h>
@@ -34,7 +34,7 @@
 
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "TileCache.hpp"
diff --git a/common/Session.hpp b/common/Session.hpp
index 736e63d..9873aad 100644
--- a/common/Session.hpp
+++ b/common/Session.hpp
@@ -22,7 +22,7 @@
 #include <Poco/StringTokenizer.h>
 #include <Poco/Types.h>
 
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "MessageQueue.hpp"
diff --git a/configure.ac b/configure.ac
index 9b6fc3d..7aa8775 100644
--- a/configure.ac
+++ b/configure.ac
@@ -32,7 +32,7 @@ AC_SUBST([LOOLWSD_VERSION_HASH])
 
 AC_DEFINE_UNQUOTED([LOOLWSD_VERSION_HASH],[["$LOOLWSD_VERSION_HASH"]],[LibreOffice On-Line git hash if present])
 
-AC_CONFIG_SRCDIR([LOOLWSD.cpp])
+AC_CONFIG_SRCDIR([wsd/LOOLWSD.cpp])
 
 AC_CONFIG_HEADERS([config.h])
 
diff --git a/kit/ChildSession.cpp b/kit/ChildSession.cpp
index c28cfbf..a0505fe 100644
--- a/kit/ChildSession.cpp
+++ b/kit/ChildSession.cpp
@@ -19,7 +19,7 @@
 #include <Poco/URI.h>
 
 #include "common/FileUtil.hpp"
-#include "LOKitHelper.hpp"
+#include "KitHelper.hpp"
 #include "Log.hpp"
 #include "Png.hpp"
 #include "Util.hpp"
diff --git a/kit/ChildSession.hpp b/kit/ChildSession.hpp
index 624090c..a392c63 100644
--- a/kit/ChildSession.hpp
+++ b/kit/ChildSession.hpp
@@ -16,8 +16,8 @@
 #include <Poco/Thread.h>
 
 #include "Common.hpp"
-#include "LOOLKit.hpp"
-#include "LOOLSession.hpp"
+#include "Kit.hpp"
+#include "Session.hpp"
 #include "LibreOfficeKit.hpp"
 
 class ChildSession;
diff --git a/kit/ForKit.cpp b/kit/ForKit.cpp
index 6763225..bccd343 100644
--- a/kit/ForKit.cpp
+++ b/kit/ForKit.cpp
@@ -33,7 +33,7 @@
 
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOOLKit.hpp"
+#include "Kit.hpp"
 #include "Log.hpp"
 #include "Unit.hpp"
 #include "Util.hpp"
diff --git a/kit/Kit.cpp b/kit/Kit.cpp
index 48902cb..a8e4e73 100644
--- a/kit/Kit.cpp
+++ b/kit/Kit.cpp
@@ -51,9 +51,9 @@
 #include "ChildSession.hpp"
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOKitHelper.hpp"
-#include "LOOLKit.hpp"
-#include "LOOLProtocol.hpp"
+#include "KitHelper.hpp"
+#include "Kit.hpp"
+#include "Protocol.hpp"
 #include "LOOLWebSocket.hpp"
 #include "LibreOfficeKit.hpp"
 #include "Log.hpp"
diff --git a/test/Makefile.am b/test/Makefile.am
index 7600797..c79d87e 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -9,7 +9,8 @@ check_PROGRAMS = test
 
 noinst_PROGRAMS = test unittest
 
-AM_CXXFLAGS = $(CPPUNIT_CFLAGS) -DTDOC=\"$(top_srcdir)/test/data\"
+AM_CXXFLAGS = $(CPPUNIT_CFLAGS) -DTDOC=\"$(top_srcdir)/test/data\" \
+	-I${top_srcdir}/common -I${top_srcdir}/wsd -I${top_srcdir}/kit
 
 noinst_LTLIBRARIES = \
         unit-timeout.la unit-prefork.la \
@@ -29,15 +30,15 @@ AM_CPPFLAGS = -pthread -I$(top_srcdir) -DBUILDING_TESTS
 wsd_sources = \
             ../common/FileUtil.cpp \
             ../common/SigUtil.cpp \
-            ../IoUtil.cpp \
-            ../Log.cpp \
-            ../LOOLKit.cpp \
-            ../LOOLProtocol.cpp \
-            ../LOOLSession.cpp \
-            ../TileCache.cpp \
-            ../MessageQueue.cpp \
-            ../Unit.cpp \
-            ../Util.cpp
+            ../common/IoUtil.cpp \
+            ../common/Log.cpp \
+            ../common/Protocol.cpp \
+            ../common/Session.cpp \
+	    ../common/MessageQueue.cpp \
+            ../kit/Kit.cpp \
+            ../wsd/TileCache.cpp \
+            ../common/Unit.cpp \
+            ../common/Util.cpp
 
 test_CPPFLAGS = -I$(top_srcdir) -DBUILDING_TESTS
 test_SOURCES = TileCacheTests.cpp integration-http-server.cpp \
diff --git a/test/TileCacheTests.cpp b/test/TileCacheTests.cpp
index 5343092..072898a 100644
--- a/test/TileCacheTests.cpp
+++ b/test/TileCacheTests.cpp
@@ -12,7 +12,7 @@
 #include <cppunit/extensions/HelperMacros.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "MessageQueue.hpp"
 #include "Png.hpp"
diff --git a/test/TileQueueTests.cpp b/test/TileQueueTests.cpp
index 964df48..dea692b 100644
--- a/test/TileQueueTests.cpp
+++ b/test/TileQueueTests.cpp
@@ -12,7 +12,7 @@
 #include <cppunit/extensions/HelperMacros.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "MessageQueue.hpp"
 #include "Util.hpp"
 
diff --git a/test/UnitFonts.cpp b/test/UnitFonts.cpp
index 60312d4..265d5b2 100644
--- a/test/UnitFonts.cpp
+++ b/test/UnitFonts.cpp
@@ -20,7 +20,7 @@
 #include <Poco/Timestamp.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "Log.hpp"
 #include "Unit.hpp"
 #include "Util.hpp"
diff --git a/test/UnitFuzz.cpp b/test/UnitFuzz.cpp
index 7454aa4..fba1d73 100644
--- a/test/UnitFuzz.cpp
+++ b/test/UnitFuzz.cpp
@@ -16,7 +16,7 @@
 
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Unit.hpp"
 #include "Util.hpp"
diff --git a/test/UnitPrefork.cpp b/test/UnitPrefork.cpp
index 9f5463b..93d4adb 100644
--- a/test/UnitPrefork.cpp
+++ b/test/UnitPrefork.cpp
@@ -24,7 +24,7 @@
 
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "Unit.hpp"
diff --git a/test/WhiteBoxTests.cpp b/test/WhiteBoxTests.cpp
index fd3814a..69faa72 100644
--- a/test/WhiteBoxTests.cpp
+++ b/test/WhiteBoxTests.cpp
@@ -13,8 +13,8 @@
 
 #include <ChildSession.hpp>
 #include <Common.hpp>
-#include <LOOLKit.hpp>
-#include <LOOLProtocol.hpp>
+#include <Kit.hpp>
+#include <Protocol.hpp>
 #include <MessageQueue.hpp>
 #include <Util.hpp>
 
diff --git a/test/helpers.hpp b/test/helpers.hpp
index 737601c..9d2c8b8 100644
--- a/test/helpers.hpp
+++ b/test/helpers.hpp
@@ -41,7 +41,7 @@
 
 #include <Common.hpp>
 #include "common/FileUtil.hpp"
-#include <LOOLProtocol.hpp>
+#include <Protocol.hpp>
 #include <LOOLWebSocket.hpp>
 #include <UserMessages.hpp>
 #include <Util.hpp>
diff --git a/test/httpcrashtest.cpp b/test/httpcrashtest.cpp
index 1bfd4f2..a4a4ff3 100644
--- a/test/httpcrashtest.cpp
+++ b/test/httpcrashtest.cpp
@@ -40,7 +40,7 @@
 #include <Common.hpp>
 #include <UserMessages.hpp>
 #include <Util.hpp>
-#include <LOOLProtocol.hpp>
+#include <Protocol.hpp>
 #include <LOOLWebSocket.hpp>
 #include "helpers.hpp"
 #include "countloolkits.hpp"
diff --git a/test/httpwserror.cpp b/test/httpwserror.cpp
index 0bd8b65..67fd235 100644
--- a/test/httpwserror.cpp
+++ b/test/httpwserror.cpp
@@ -22,7 +22,7 @@
 #include <cppunit/extensions/HelperMacros.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "helpers.hpp"
 #include "countloolkits.hpp"
diff --git a/test/httpwstest.cpp b/test/httpwstest.cpp
index 93a024b..042c6d3 100644
--- a/test/httpwstest.cpp
+++ b/test/httpwstest.cpp
@@ -40,7 +40,7 @@
 #include <cppunit/extensions/HelperMacros.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Png.hpp"
 #include "UserMessages.hpp"
diff --git a/tools/Connect.cpp b/tools/Connect.cpp
index 5b550dd..d45b243 100644
--- a/tools/Connect.cpp
+++ b/tools/Connect.cpp
@@ -41,7 +41,7 @@
 #include <Poco/Version.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "Util.hpp"
diff --git a/tools/KitClient.cpp b/tools/KitClient.cpp
index 06d0a33..a587a28 100644
--- a/tools/KitClient.cpp
+++ b/tools/KitClient.cpp
@@ -27,7 +27,7 @@
 #include <Poco/URI.h>
 #include <Poco/Util/Application.h>
 
-#include "LOKitHelper.hpp"
+#include "KitHelper.hpp"
 #include "Png.hpp"
 #include "Util.hpp"
 
diff --git a/tools/Tool.cpp b/tools/Tool.cpp
index b658108..437fe1d 100644
--- a/tools/Tool.cpp
+++ b/tools/Tool.cpp
@@ -40,7 +40,7 @@
 #include <Poco/Util/OptionSet.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "Util.hpp"
 
 #include <Poco/Util/Application.h>
diff --git a/wsd/Admin.cpp b/wsd/Admin.cpp
index 445fa1e..62ee9b2 100644
--- a/wsd/Admin.cpp
+++ b/wsd/Admin.cpp
@@ -31,8 +31,8 @@
 #include "Common.hpp"
 #include "FileServer.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
-#include <LOOLWebSocket.hpp>
+#include "Protocol.hpp"
+#include "LOOLWebSocket.hpp"
 #include "LOOLWSD.hpp"
 #include "Log.hpp"
 #include "Storage.hpp"
diff --git a/wsd/AdminModel.cpp b/wsd/AdminModel.cpp
index 62d4ab7..05df3d7 100644
--- a/wsd/AdminModel.cpp
+++ b/wsd/AdminModel.cpp
@@ -19,7 +19,7 @@
 #include <Poco/StringTokenizer.h>
 #include <Poco/URI.h>
 
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "Unit.hpp"
diff --git a/wsd/ClientSession.cpp b/wsd/ClientSession.cpp
index 00a0a3f..cedbc5e 100644
--- a/wsd/ClientSession.cpp
+++ b/wsd/ClientSession.cpp
@@ -20,8 +20,8 @@
 
 #include "Common.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
-#include "LOOLSession.hpp"
+#include "Protocol.hpp"
+#include "Session.hpp"
 #include "LOOLWSD.hpp"
 #include "Log.hpp"
 #include "PrisonerSession.hpp"
diff --git a/wsd/ClientSession.hpp b/wsd/ClientSession.hpp
index 7b7740c..e958f34 100644
--- a/wsd/ClientSession.hpp
+++ b/wsd/ClientSession.hpp
@@ -10,7 +10,7 @@
 #ifndef INCLUDED_CLIENTSSESSION_HPP
 #define INCLUDED_CLIENTSSESSION_HPP
 
-#include "LOOLSession.hpp"
+#include "Session.hpp"
 #include "MessageQueue.hpp"
 
 #include <Poco/URI.h>
diff --git a/wsd/DocumentBroker.cpp b/wsd/DocumentBroker.cpp
index a71d405..b814d35 100644
--- a/wsd/DocumentBroker.cpp
+++ b/wsd/DocumentBroker.cpp
@@ -22,7 +22,7 @@
 #include "Admin.hpp"
 #include "ClientSession.hpp"
 #include "Exceptions.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "LOOLWSD.hpp"
 #include "Log.hpp"
 #include "PrisonerSession.hpp"
diff --git a/wsd/LOOLWSD.cpp b/wsd/LOOLWSD.cpp
index e6a5f27..dd8df4b 100644
--- a/wsd/LOOLWSD.cpp
+++ b/wsd/LOOLWSD.cpp
@@ -101,8 +101,8 @@
 #include "FileServer.hpp"
 #include "common/FileUtil.hpp"
 #include "IoUtil.hpp"
-#include "LOOLProtocol.hpp"
-#include "LOOLSession.hpp"
+#include "Protocol.hpp"
+#include "Session.hpp"
 #include <LOOLWebSocket.hpp>
 #include "Log.hpp"
 #include "PrisonerSession.hpp"
diff --git a/wsd/LOOLWebSocket.hpp b/wsd/LOOLWebSocket.hpp
index b4956ce..766790e 100644
--- a/wsd/LOOLWebSocket.hpp
+++ b/wsd/LOOLWebSocket.hpp
@@ -15,7 +15,7 @@
 #include <Poco/Net/WebSocket.h>
 
 #include <Common.hpp>
-#include <LOOLProtocol.hpp>
+#include <Protocol.hpp>
 #include <Log.hpp>
 
 /// WebSocket that is thread safe, and handles large frames transparently.
diff --git a/wsd/PrisonerSession.cpp b/wsd/PrisonerSession.cpp
index bb352cc..8fea0e1 100644
--- a/wsd/PrisonerSession.cpp
+++ b/wsd/PrisonerSession.cpp
@@ -17,8 +17,8 @@
 #include <Poco/URIStreamOpener.h>
 
 #include "Common.hpp"
-#include "LOOLProtocol.hpp"
-#include "LOOLSession.hpp"
+#include "Protocol.hpp"
+#include "Session.hpp"
 #include "LOOLWSD.hpp"
 #include "Log.hpp"
 #include "ClientSession.hpp"
diff --git a/wsd/PrisonerSession.hpp b/wsd/PrisonerSession.hpp
index a52dfc8..1518ab4 100644
--- a/wsd/PrisonerSession.hpp
+++ b/wsd/PrisonerSession.hpp
@@ -10,7 +10,7 @@
 #ifndef INCLUDED_PRISONERSESSION_HPP
 #define INCLUDED_PRISONERSESSION_HPP
 
-#include "LOOLSession.hpp"
+#include "Session.hpp"
 
 class DocumentBroker;
 class ClientSession;
diff --git a/wsd/QueueHandler.hpp b/wsd/QueueHandler.hpp
index 3f330e6..445b91e 100644
--- a/wsd/QueueHandler.hpp
+++ b/wsd/QueueHandler.hpp
@@ -9,8 +9,8 @@
 
 #include <Poco/Runnable.h>
 
-#include "LOOLProtocol.hpp"
-#include "LOOLSession.hpp"
+#include "Protocol.hpp"
+#include "Session.hpp"
 #include "MessageQueue.hpp"
 #include "Util.hpp"
 
diff --git a/wsd/TileCache.cpp b/wsd/TileCache.cpp
index 41d9bd0..0ce257f 100644
--- a/wsd/TileCache.cpp
+++ b/wsd/TileCache.cpp
@@ -33,7 +33,7 @@
 #include "ClientSession.hpp"
 #include "Common.hpp"
 #include "common/FileUtil.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 #include "Unit.hpp"
 #include "Util.hpp"
 
diff --git a/wsd/TileDesc.hpp b/wsd/TileDesc.hpp
index 8ba44bd..111c509 100644
--- a/wsd/TileDesc.hpp
+++ b/wsd/TileDesc.hpp
@@ -18,7 +18,7 @@
 #include <Poco/StringTokenizer.h>
 
 #include "Exceptions.hpp"
-#include "LOOLProtocol.hpp"
+#include "Protocol.hpp"
 
 /// Tile Descriptor
 /// Represents a tile's coordinates and dimensions.
