--- poco-1.12.4-all/Foundation/src/Thread_POSIX.cpp	2022-10-31 18:44:32.000000000 +0100
+++ poco-1.12.4-all/Foundation/src/Thread_POSIX.cpp	2022-11-09 18:20:41.701346868 +0100
@@ -67,7 +67,7 @@
 {
 #if (POCO_OS == POCO_OS_MAC_OS_X)
 	pthread_setname_np(threadName.c_str()); // __OSX_AVAILABLE_STARTING(__MAC_10_6, __IPHONE_3_2)
-#else
+#elif !defined(__EMSCRIPTEN__)
 	if (pthread_setname_np(thread, threadName.c_str()) == ERANGE && threadName.size() > 15)
 	{
 		std::string truncName(threadName, 0, 7);
@@ -272,7 +272,7 @@
 
 long ThreadImpl::currentOsTidImpl()
 {
-#if POCO_OS == POCO_OS_LINUX
+#if POCO_OS == POCO_OS_LINUX && !defined(__EMSCRIPTEN__)
     return ::syscall(SYS_gettid);
 #elif POCO_OS == POCO_OS_MAC_OS_X
     return ::pthread_mach_thread_np(::pthread_self());
--- poco-1.12.4-all/Net/include/Poco/Net/Net.h
+++ poco-1.12.4-all/Net/include/Poco/Net/Net.h
@@ -132,7 +132,7 @@
 #endif
 
 
-#if (POCO_OS == POCO_OS_LINUX) || (POCO_OS == POCO_OS_WINDOWS_NT)
+#if ((POCO_OS == POCO_OS_LINUX) || (POCO_OS == POCO_OS_WINDOWS_NT)) && !defined(__EMSCRIPTEN__)
 	#define POCO_HAVE_FD_EPOLL 1
 #endif
 
--- poco-1.12.4-all/build/config/Linux
+++ poco-1.12.4-all/build/config/Linux
@@ -63,7 +63,7 @@
 #
 # System Specific Flags
 #
-SYSFLAGS = -D_XOPEN_SOURCE=600 -D_REENTRANT -D_THREAD_SAFE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -DPOCO_HAVE_FD_EPOLL
+SYSFLAGS = -D_XOPEN_SOURCE=600 -D_REENTRANT -D_THREAD_SAFE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE
 
 #
 # System Specific Libraries
